reference/configuration.html.markerb

```diff
@@ -70,6 +70,23 @@ kill_timeout = 120
 
 <section class="warning">Note: `kill_timeout` settings should be considered best-effort, and your app needs to be prepared to handle shorter stopping times</section>
 
+**How the shutdown sequence works**
+These options come into play during controlled shutdowns such as `fly deploy` or when a machine stops due to `auto_stop_machines`. The sequence looks like this:
```

Comment:
`fly machine stop` also does a shutdown sending the two signals and respecting kill_timeout.